pypeit/spectrographs/gemini_gmos.py

```diff
@@ -196,7 +196,7 @@ def get_rawimage(self, raw_file, det):
         Returns:
             tuple:
                 raw_img (np.ndarray) -- Raw image for this detector
-                hdu (astropy.io.fits.HDUList)
+                headarr (list of headers)
```

Comment:
As above, I really wish to keep this as `hdu`

Reply:
Done!